nextflow.config

```diff
@@ -21,9 +20,6 @@ params {
     // Annotation options
     run_annotation_tool             = 'prodigal'
 
-    // Prodigal Annotation
-    prodigal_output_format           = 'gff'
-
```

Comment:
Yes, no other tools except antiSMASH need input from the `prodigal_output_format` variable. We are now forcing GFF because this works with all datasets (tested so far). The other possible format (GBK) is not accepted by antiSMASH if it comes from prodigal (prokka is fine though, what a mess). Anyway, it makes the code cleaner to force GFF instead of if-else-ing around in the `bgc.nf` subworkflow.